disnake/message.py

```diff
@@ -152,8 +152,16 @@ async def _edit_handler(
 
     if embed is not MISSING:
         embeds = [embed] if embed else []
+        if isinstance(embed, Embed) and embed._files:
+            embed._files.clear()
```

Comment:
these two lines are not necessary since this sets the `embeds` variable which is processed below :)

Reply:
oh true lmaoo :'(